### DISCUSSION TITLE
Question about the Sniping

### DISCUSSION THREAD

**Discussion by CloudyX2021** [marioparaschiv/nitro-sniper#236] [Q&A] [answered]
I have made an alt and started Nitro Sniper in repl.it, after some hours I checked it back and I still haven't got any nitro or even expired or claimed code.

**Comment by MrD4rio** [comment, marked as answer]
Sniping could require several hours or even days. All depends on the number of servers you're in. If you're worried about the sniper not working, you can test the detection of gift codes by simply sending a fake/already redeemed one in a test server where your sniping alt is there.

**Reply by breamann** [reply]
hello! i tried this and it still didn't give me a response. i have it ticked "true" on invalid codes to send a thing inside of the console and an @everyone on the webhook and over and over again I haven't seen a response.

any fix to this?

**Reply by MrD4rio** [reply]
Have you tried with various different codes? Consider that sending always the same code might result into it not even being shown in console as it's a duped code. Also make sure to recheck your config.
